src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.js

```diff
@@ -44,9 +44,21 @@ const EditButton = styled(Button)`
   padding: ${themeSpacing(0, 1.5)};
 `;
 
+const getCategoryName = ({ name, departments }) => {
+  const departmensStringList = departments?.length ? ` (${departments.map(({ code }) => code).join(',')})` : '';
+  return `${name}${departmensStringList}`;
+};
+
```

Comment:
Can you add a test case that verifies the addition of the concatenated department names to the subcategory?

Reply:
done